Exercise_2_Currency_Ex/exchange.md

```diff
@@ -6,12 +6,12 @@ Placing the pseudocode in this document will also allow me to keep the codebase
 
 ## Pseudocode
 
-`def estimate_value(budget, exchange_rate):`
-`def get_change(budget, exchanging_value):`
-`def get_value(denomination, number_of_bills):`
-`def get_number_of_bills(budget, denomination):`
+`def estimate_value(budget, exchange_rate):`  
+`def get_change(budget, exchanging_value):`  
+`def get_value(denomination, number_of_bills):`  
+`def get_number_of_bills(budget, denomination):`  
```

Comment:
you had the correct syntax for the inline code with just a single opening/closing back tick for the line as a whole

easier still would be to simply put all the function names in a code fence

````
```python
def func1():
    pass
```
````

Reply:
Updated in the latest PR, thanks!